packages/dashboards/tests/integration/helpers/reportlize-test.js

```diff
@@ -0,0 +1,47 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+const widgetModel = {
+  author: 'navi',
+  request: {
+    clone() {
+      return {
+        test: 'foo'
+      };
+    }
+  },
+  toJSON() {
+    return {
+      title: 'test',
+      visualization: {
+        type: 'line-chart'
+      },
+    }
+  }
+}
+
+moduleForComponent('helper:reportlize', 'Integration | Helper | reportlize', {
+  integration: true,
+});
+
+test('onModelChange', function (assert) {
+  assert.expect(2);
+
+  this.set('model', widgetModel);
+  this.render(hbs`{{get (reportlize model) 'request.test'}}`)
```

Comment:
Can we fix the typo in the helper name? I think it should be `reportize` instead of `reportlize`. Without the `l`.